src/components/Avatar/Avatar.stories.ts

```diff
@@ -0,0 +1,57 @@
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import type { StoryObj, Meta } from '@storybook/react';
+import type React from 'react';
+
+import { Avatar } from './Avatar';
+
+export default {
+  title: 'Components/Avatar',
+  component: Avatar,
+  args: {
+    size: 'md',
+    shape: 'circle',
+    variant: 'icon',
+  },
+  parameters: {
+    badges: [BADGE.BETA],
+    layout: 'centered',
+  },
+} as Meta<Args>;
+
+type Args = React.ComponentProps<typeof Avatar>;
+
+export const Default: StoryObj<Args> = {
+  args: {},
+};
+
+export const Small: StoryObj<Args> = {
+  args: {
+    size: 'sm',
+  },
+};
+
+export const Medium: StoryObj<Args> = {
+  args: {
+    size: 'md',
+  },
+};
+
+export const Large: StoryObj<Args> = {
+  args: {
+    size: 'lg',
+  },
+};
+
+export const UsingImage: StoryObj<Args> = {
+  args: {
+    variant: 'image',
+    src: 'http://placekitten.com/200/200',
```

Comment:
🐈👍

Reply:
I'll have to remove this probably :( the image it returns might not be stable, and we'd be hitting that site a lot. What I may do is include some generic inline svg or something